project/npda/management/commands/upload_csv.py

```diff
@@ -0,0 +1,46 @@
+from asgiref.sync import async_to_sync
```

Comment:
I wrote this for performance profiling as you can put cProfile in front of it easily (https://stackoverflow.com/questions/582336/how-do-i-profile-a-python-script).